frb_cors/README.md

```diff
@@ -0,0 +1,3 @@
+# frb_cors
+
+A simple static web server for serving Flutter WASM applications.
```

Comment:
I'm not good with names so I'll leave it to you 😅

Reply:
I realized another thing: Can this be moved into the existing `frb_dart` package? Since it seems only to be a dart file in `bin/`. If so users do not need to be confused with one extra package.